src/server/api/controllers/users.controller.js

```diff
@@ -18,6 +19,26 @@ const createUser = async (body) => {
   };
 };
 
+const getUsers = async () => knex('users');
+
+const getUserById = async (id) => {
+  if (!id) {
```

Comment:
Hi Milton, I don really understand how to use parseInt here. what do you mean by the following:

const user_id = parseInt(query_user_id);
if (isNaN(user_id) || user_id < 1) { ...

Reply:
The function `getUserById` takes an argument, called `id`, which likely is of type "string" since all values in HTTP are strings.
So to be safe, you should first convert it from a string to a number, by using `parseInt`. 
`parseInt` returns either a number or the constant `NaN`, so you can then update the if statement to check if you got back the constant `NaN` (in which case, the input parameter is not valid at all since it's not a number) or if the value is below 1 (IDs in the database should never be smaller than 1).

Does it make more sense now? :)